src/styles/main.scss

```diff
@@ -24,9 +20,7 @@
 @import 'contact';
 @import 'legal';
 @import 'form';
-
 /* Temporary */
 @import 'coming-soon';
-
 /* Outdate Browsers*/
-@import '../../bower_components/outdated-browser/outdatedbrowser/outdatedbrowser.scss';
+// @import '../../bower_components/outdated-browser/outdatedbrowser/outdatedbrowser.scss';
```

Comment:
Sooo I understand you uncommented this because it broke your SASS. Now that you have done `bower install` this should work. Please uncomment this and do a `grunt build`, see if that works.